packages/statemanager/src/statelessVerkleStateManager.ts

```diff
@@ -112,14 +115,19 @@ export class StatelessVerkleStateManager implements StateManagerInterface {
 
     this._caches = opts.caches
 
-    this.keccakFunction = opts.common?.customCrypto.keccak256 ?? keccak256
-
-    this._debug = debugDefault('statemanager:verkle:stateless')
+    if (opts.common.isActivatedEIP(6800) === false) {
+      throw new Error('EIP-6800 required for verkle state management')
```

Comment:
```suggestion
      throw new Error('EIP-6800 required for stateless verkle state management')
```

This is the stateless verkle state manager, so I don't think this message is completely correct?